crates/bevy_sprite/src/sprite.rs

```diff
@@ -3,6 +3,9 @@ use bevy_math::{Rect, Vec2};
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
 use bevy_render::color::Color;
 
+/// Specifies the rendering properties of a sprite.
+///
+/// [`SpriteBundle`](crate::bundle::SpriteBundle) is a bundle for common sprite rendering use cases.
```

Comment:
```suggestion
/// This is commonly used as a component within [`SpriteBundle`](crate::bundle::SpriteBundle).
```

We're talking about `Sprite` here, not `SpriteBundle`.